### ISSUE TITLE
Support for input of type "date"

### ISSUE THREAD

**Issue by s427** [felicegattuso/datedropper#78]
I noticed this plug-in first tests if the type of the input is "text":

`if` ($(this).is('input') && $(this).attr('type') == "text") {

I suppose the reason this plug-in doesn't support the "date" input is that those fields normally provide their own date picker mechanism, so that would be redundant (or even cause conflict).

The problem is that support for the "date" input is still far from perfect. In particular, Firefox and IE don't support it yet, so we need a fallback mechanism for them, and this plug-in is perfect for that.

It is possible to test for the support of this input type via the "type" property of the input (whose value is "text" if the type is not recognized) (see: http://stackoverflow.com/a/18021130/357546)

So I suggest changing the above condition into:

```
if (
    $(this).is('input') && (
        $(this).attr('type') == "text" || (
            $(this).attr('type') == "date" &&
            $(this).prop('type') != 'date'
        )
    )
) {

```
In other words:
if the element is a text input, or a date input whose type property is not "date" (i.e. non-recognized date input).

That way the plug-in can be used safely as a fallback mechanism for date inputs when they are not recognized by the browser.

**Comment by CedricReichenbach** [comment]
I just discovered this issue too. Surprizingly, timedropper works with input type "time".

For my use case, I'm trying to use built-in support and use timedropper/datedropper as fallback. To do si, I set input type to `date`, then check `input.prop('type')`. If this returns `text`, that means the `date` input type is not supported and I fall back to datedropper (but the `type` *attribute* is obviously still `date`).
